UIKitLearnings/UIKitLearnings/Animations/AnimationPlayViewController.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  AnimationPlayViewController.swift
+//  UIKitLearnings
+//
+//  Created by Saurabh Verma on 04/06/24.
+//
+
+import UIKit
+
+class AnimationPlayViewController: UIViewController {
+
+    var animationOption: AnimationsOption!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        if let animatingView = getAnimationView() {
+            self.view.addSubview(animatingView)
+        }
+    }
+    
+    func getAnimationView() -> UIView? {
```

Comment:
At present animation gets started as view appears, there should be way to toggle animation once view is loaded. Add a TODO to create this.